README.md

```diff
@@ -1 +1,50 @@
-# bfx-facs-hypercore-logger
\ No newline at end of file
+# bfx-facs-hypercore-logger
+
+## Getting started
+Install [Hypercore CLI](https://hypercore-protocol.org/) and [Hypercore Logs](https://github.com/bitfinexcom/hypercore-logs)
+```
+npm install -g @hyperspace/cli 
```

Comment:
hmm why do we need hyperspace cli here? I think hypercore feed could run without it